src/Common/QueryProfiler.cpp

```diff
@@ -57,7 +57,7 @@ namespace
 
         auto saved_errno = errno;   /// We must restore previous value of errno in signal handler.
 
-#if defined(OS_LINUX)
+#if defined(OS_LINUX) && false //asdqwe
```

Comment:
Why was it merged with these changes?

Reply:
Sorry. https://github.com/ClickHouse/ClickHouse/pull/67152